app/uk/gov/hmrc/vulnerabilities/persistence/VulnerabilitiesTimelineRepository.scala

```diff
@@ -34,13 +39,17 @@ class VulnerabilitiesTimelineRepository @Inject()(
 ) extends PlayMongoRepository(
   collectionName = "vulnerabilitiesTimeline",
   mongoComponent = mongoComponent,
-  domainFormat   = ServiceVulnerability.mongoFormat,
+  domainFormat   = TimelineEvent.mongoFormat,
   indexes        = Seq(
-    IndexModel(compoundIndex(descending("weekBeginning"), descending("service"), descending("id")), IndexOptions().unique(true)),
-    IndexModel(ascending("weekBeginning"), IndexOptions().expireAfter(2 * 365, TimeUnit.DAYS))
+    IndexModel(compoundIndex(descending("weekBeginning"), descending("service"), descending("id")),IndexOptions().unique(true).background(true)),
+    IndexModel(Indexes.ascending("weekBeginning"),IndexOptions().expireAfter(2 * 365, TimeUnit.DAYS).background(true)),
```

Comment:
CLARIFICATION: Can we make the expiration configurable?

Reply:
Agreed on slack that its fine to leave in code, as seems to be how we do it everywhere else